src/apps/common/submodules/chooseModel/chooseModel.js

```diff
@@ -39,19 +39,26 @@ define(function(require) {
 					brands: []
 				},
 				families,
-				models;
-
+				models,
+				prefix = ['vvx'];
```

Comment:
I think the line break is needed just to indicate that there is a new block of code

Reply:
changed